cmd/cagent/main.go

```diff
@@ -276,19 +276,21 @@ func main() {
 
 			if err != nil {
 				fmt.Printf("Already running\n")
```

Comment:
should we maybe terminate if already running?

Reply:
I think there is no need to terminate here. It will only show the how-to-stop message(it could be useful) and the logs location